test-site/src/components/MarkdownExampleCard.tsx

```diff
@@ -0,0 +1,42 @@
+import { CardProps, useCardAnalyticsCallback, useCardFeedbackCallback } from '@yext/search-ui-react';
+import { useCallback } from 'react';
+import '../styles/resetStyles.css'
+import MDEditor from "@uiw/react-md-editor";
+
+
+
+const builtInCssClasses = {
+    container: 'flex flex-col justify-between border border-gray-200 rounded-lg mb-4 p-4 shadow-sm',
+    header: 'flex text-neutral-dark',
+    title: 'text-lg font-medium',
+    thumbsFeedbackContainer: 'flex justify-end mt-4 text-sm text-gray-500 font-medium',
+};
+
+
+const markdownFieldName  = "c_markdownData";
+
+interface CustomRawDataType {
+    name: string,
+    description: string,
```

Comment:
same here

Reply:
(same comment as above): This is used to define the type of prop used in the exported function. Without it, typescript seems to give errors when we try to access the user-specified field name